docs/checklist.md

```diff
@@ -27,6 +27,37 @@ Comes in three sections. You need varying degrees of access depending on where y
 - ChecklistProvider: Wraps around the carousel so that the toggle buttons in the secondary menu know when the checklist is open and the other popup menus there (help center) can respect it. 
 - ChecklistCountProvider, ChecklistCategoryProvider: Wrap around various parts of the checklist itself to provide counts of issues.
 
+## Roles
+
+Depending on the user's `role`, the user may or may not be able to fix specific checklist items. A checklist card will be hidden if the user is unable to fix the issue.
+
+This is outlined in the table below:
+
+|Story issue/card|Role(s) necessary to resolve the issue|
+|--|--|
+|Story missing title|`edit_web-stories`|
```

Comment:
Listing `edit_web-stories` here is redundant.

If you don't have the `edit_web-stories` capability you can't even open the story editor.